### PR TITLE
Minor fixes on drives listing and formatting

### PR DESCRIPTION
**Pull request by Praveenrajmani**

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -272,6 +272,7 @@ func (c *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 
 	drive, err := matchDrive()
 	if err != nil {
+		utils.JSONifyAndLog(req.GetAccessibilityRequirements())
```
Comment (Praveenrajmani) [inline pkg/controller/controller.go:275]: Bringing back this log, as this will be useful to check if the volumes were asked on wrong nodes. (in topology constraint failure cases)

File: cmd/kubectl-direct_csi/drives_list.go [hunk 2]
```diff
@@ -137,11 +137,12 @@ func listDrives(ctx context.Context, args []string) error {
 		},
 		Items: filteredDrives,
 	}
-	if printer != nil {
+	if yaml || json {
```
Comment (wlan0) [inline cmd/kubectl-direct_csi/drives_list.go:140]: 👍🏾

File: pkg/controller/controller.go (hunk 1 above)
Comment (Praveenrajmani) [reply pkg/controller/controller.go:275]: I will revert this change and bring these changes as a separate PR which can take in logging improvisations.

File: pkg/utils/utils.go [hunk 3]
```diff
@@ -17,14 +17,25 @@
...
 	"reflect"
 	"strings"
 
 	directcsi "github.com/minio/direct-csi/pkg/apis/direct.csi.min.io/v1beta2"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/klog"
 )
 
+func JSONifyAndLog(val interface{}) {
```
Comment (Praveenrajmani) [inline pkg/utils/utils.go:31]: Will revert this change

**Review by wlan0 — APPROVED** [review]
